lib/checks/navigation/identical-links-same-purpose.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * Note: `identical-links-same-purpose-after` fn, helps reconcile the results
+ */
+const { text } = axe.commons;
+
+/**
+ * Note:
+ * Firefox does not respect map > area as visible,
+ * even when used by a img[usemap], hence the usage of `includeHidden` flag,
+ * in the accessible name computation
+ */
+const accText = text.accessibleTextVirtual(virtualNode, {
+	includeHidden: true
+});
+const name = curateText(accText, {
+	emoji: true,
+	nonBmp: true,
+	punctuations: true,
+	sanitize: true,
+	lowercase: true
+});
+if (!name) {
+	return undefined;
+}
+
+/**
+ * Set `data` and `relatedNodes` for use in `after` fn
+ */
+const afterData = {
+	name,
+	parsedResource: getParsedResource(node, 'href')
```

Comment:
You're now parsing the HREF in `<div href="foo" role="link">`. Probably shouldn't do that.